packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-card.spec.tsx

```diff
@@ -148,7 +152,9 @@ describe('JurisdictionCard', () => {
         expect(screen.getByText('British Virgin Islands')).toBeInTheDocument();
         expect(screen.getByText('Assets')).toBeInTheDocument();
         expect(screen.getByText('170+')).toBeInTheDocument();
-        expect(screen.getByText('Forex, Stocks, Stock indices, Commodities, and Cryptocurrencies')).toBeInTheDocument();
+        expect(
+            screen.getByText('Forex, stocks, stock indices, commodities, cryptocurrencies, and ETFs')
```

Comment:
Are the capital or small letters based on the design because I can see that `Stock indices` is changed to `stock indices`